mlflow/pipelines/cli.py

```diff
@@ -93,3 +93,22 @@ def inspect(step, profile):
     nothing is displayed.
     """
     Pipeline(profile=profile).inspect(step)
+
+
+@commands.command(short_help=("Get the location of an artifact."))
+@click.option(
+    "--artifact",
+    "-a",
+    type=click.STRING,
+    default=None,
+    required=False,
```

Comment:
`artifact` is not required? What's the expected behavior for None?

Reply:
good catch. will fix this.